### PR TITLE
Added SMT PLCC footprints

### PR DESCRIPTION
**Pull request by SchrodingersGat**

### CODE REVIEW

**Comment by jkriege2** [comment]
These are SMT PLCC sockets, right? ... as shown here: http://cdn-reichelt.de/documents/datenblatt/C100/SMDPLCC20_SMDPLCC28_SMDPLCC32_SMDPLCC44%23MPE.pdf

If so I would rename to `PLCC-Socket-xy_...` to make that clear ... and could you tweak your script to also generate land patterns to directly solder PLCCs to the PCB (maybe you can use the images there: http://www.beckwithelectronics.com/ARIES/1110125.htm as basis for the footprint)?

cu
JAN

**Comment by SchrodingersGat** [comment]
Updated!

**Comment by jkriege2** [comment]
Hi! In the DIP-repo we used `_SMDSocket` ... should we use the same naming here?

**Comment by SchrodingersGat** [comment]
Done. I have also changed the THT versions [here](https://github.com/KiCad/Housings_LCC.pretty/pull/4)

**Comment by jkriege2** [comment]
A bit nitpicking, but for DIP we had `SMDSocket` as one word, no hyphen ... shoulld we do it here the same?

**Comment by SchrodingersGat** [comment]
I think that THTSocket and SMDSocket should actually be separated into e.g. THT-Socket:

a) This is easier to read
b) It separates the words and functions

Would you agree? I find `THTSocket` for example a little hard to parse

**Comment by jkriege2** [comment]
Fine with me ... since I do a lot of programming THTSocket and SMDSocket is easy for me to read ;-)
